src/Resources/Group.php

```diff
@@ -2,10 +2,21 @@
 
 namespace DigitalCreative\CollapsibleResourceManager\Resources;
 
+use Illuminate\Http\Request;
+
 class Group extends TopLevelResource
 {
     protected $type = 'group';
 
+    public function authorizedToSee(Request $request)
```

Comment:
@milewski this function seems to introduce a breaking change:

```
DigitalCreative\CollapsibleResourceManager\Resources\Group::DigitalCreative\CollapsibleResourceManager\Resources\{closure}(): 
Argument #1 ($resource) must be of type string, DigitalCreative\CollapsibleResourceManager\Resources\NovaResource given,

````

Reply:
Can you send me an example if your configuration file

Reply:
Nevermind I found the issue